app/templates/formbuilder/edit/code.hbs

```diff
@@ -1,41 +1,39 @@
 {{page-title "Code editor"}}
 
 <div class="au-c-body-container au-u-margin-bottom-none">
-  <AuBodyContainer
-    @scroll={{true}}
-  >
+  <AuBodyContainer @scroll={{true}}>
     <AuContent
       @skin="small"
-      {{editor this.formCode (perform this.handleCodeChange)}}>
-    </AuContent>
+      {{editor this.formCode (perform this.handleCodeChange)}}
+    />
   </AuBodyContainer>
   <div class>
-    <AuButton 
+    <AuButton
       class="au-u-background-gray-100 border"
       @skin="naked"
       @width="block"
       {{on "click" this.toggleOpenCloseConsoleState}}
     >
-      {{t "codeConsole.buttonNotification"}}: {{this.warnings.length}}
+      {{t "codeConsole.buttonNotification"}}:
+      {{this.consoleMessages.length}}
     </AuButton>
   </div>
   {{#unless this.consoleClosed}}
-  <AuBodyContainer class="code-edit-console__container">
-    <AuBodyContainer 
-      class="code-edit-console"
-      @scroll={{true}}
-    >
-      {{#each this.warnings as |warning|}}
-        <AuContent
-          class="code-edit-warning"
-        >
-          <AuIcon @icon="alert-triangle" @size="large" @alignment="left"/>
-          <p class="au-u-bold">Subject: </p>
-          <p>{{warning.subject}}</p>
-          <p class="au-u-bold">{{warning.message}}</p>
-        </AuContent>
-      {{/each}}
+    <AuBodyContainer class="code-edit-console__container">
+      <AuBodyContainer class="code-edit-console" @scroll={{true}}>
+        {{#each this.consoleMessages as |message|}}
+          <AuContent class={{message.severity.class}}>
+            <AuIcon
+              class={{message.severity.iconClass}}
+              @icon={{message.severity.icon}}
+              @size="large"
+              @alignment="left"
+            />
+            <p>{{message.subject}}</p>
+            <p class="au-u-bold">{{message.content}}</p>
+          </AuContent>
+        {{/each}}
+      </AuBodyContainer>
```

Comment:
Will look like this now 
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/c9230238-b997-40a2-a3f8-1a0236c77c5a)

we should definitly think about what we want to show in here and how we want to show it  as we cannot change the format everytime we have a new message to display.